documentation/plain_rules.md

```diff
@@ -92,6 +125,17 @@ Please, be carefull with using non-ascii characters in the EPL syntax. It will p
 information on how to scape characters at
 [Esper site](http://esper.espertech.com/release-6.1.0/esper-reference/html/event_representation.html#eventrep-properties-escaping)
 
+## No signal conditions
+
+The no signal condition is specified in the `nosignal` configuration element, which is an object with the following fields:
+
+-   checkInterval: _optional_ or _mandatory_ ???,time in minutes for checking the attribute
```

Comment:
checkInterval is recommended to be at least double of reportInterval. But we should warn that a very demanding value of checkInterval could impact on performance.

Reply:
Fixed in 
e2f779d